tests/manage/pv_services/test_ceph_daemon_kill_during_resource_deletion.py

```diff
@@ -304,11 +303,12 @@ def test_ceph_daemon_kill_during_pod_pvc_deletion(
         log.info("Verified: PVs are deleted.")
 
         # Verify PV using ceph toolbox. Image/Subvolume should be deleted.
+        pool_name = default_ceph_block_pool()
         for pvc_name, uuid in pvc_uuid_map.items():
             if interface == constants.CEPHBLOCKPOOL:
                 ret = verify_volume_deleted_in_backend(
                     interface=interface, image_uuid=uuid,
-                    pool_name=sc_obj.ceph_pool.name
+                    pool_name=pool_name
```

Comment:
If its verifying the specific image then it looks good, Once I have the cluster up I can quickly verify as well.